docs/File-Formats.md

```diff
@@ -125,6 +125,21 @@ The first four rows of the clinical data file contain tab-delimited metadata abo
 - Row 2: **The attribute Descriptions**: Long(er) description of each clinical attribute
 - Row 3: **The attribute Datatype**: The datatype of each clinical attribute (must be one of:  STRING, NUMBER, BOOLEAN)
 - Row 4: **The attribute Priority**: A number which indicates the importance of each attribute.  In the future, higher priority attributes will appear in more prominent places than lower priority ones on relevant pages (such as the [Study View](http://www.cbioportal.org/study?id=brca_tcga)). A lower number indicates a higher priority.
+    ```
+    To promote certain chart in study view, please increase priority in the database to certain number. The higher the score, the higher priority it will be displayed in the study view.
```

Comment:
Maybe remove the reference to "database" here, since this page is just about documenting the file format.